Makefile

```diff
@@ -375,7 +375,25 @@ $(eval $(call build-package,protobuf,3.21.12-r0))
 $(eval $(call build-package,protobuf-c,1.4.1-r0))
 $(eval $(call build-package,fstrm,0.6.1-r0))
 $(eval $(call build-package,bind,9.18.11-r0))
-$(eval $(call build-package,ruby-bundler,2.4.3-r0))
+# Ruby Ecosystem
+$(eval $(call build-package,ruby-3.0,3.0.4-r3))
```

Comment:
The previous ruby-3.0/3.1/3.2 entries need to be removed.

Reply:
Ah good catch! Must have snuck back in when I fixed the merge conflicts